example3 Kopie/shapes/python/areaCalculator.py

```
@@ -0,0 +1,74 @@
+from abc import ABC, abstractmethod
+import math
+import json
+
+class Shape(ABC):
+    @abstractmethod
+    def area(self):
+        pass
+
+class AreaCalculator:
+    def __init__(self, shapes=[]) -> None:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)